app/components/categories/card_component.rb

```diff
@@ -1,7 +1,7 @@
 class Categories::CardComponent < ViewComponent::Base
   attr_reader :title, :description, :path, :heading_tag
 
-  def initialize(card:, heading_tag: "h2")
+  def initialize(card:, heading_tag: "h3")
```

Comment:
It looks this has affected the category pages @garethwilliamsdesign 

For example, on Train to be a teacher category page:

**Production - top category cards are h2s**
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/0bf58ade-4ff3-4f62-8d51-b46f01858476)

**Review app - top category cards are h3**
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/1b674450-6a47-491e-911a-d864eacff855)

Let's discuss the best way to address this - it might be that we need to add an h2 subcategory heading on the top of the Train to be a teacher page, like there is on Is teaching right for me, as this changes the card level to be h3:

**Production**
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/56e707f7-8e8a-4fa8-a881-893b741cc0b3)